src/server/services/location.ts

```diff
@@ -34,7 +35,8 @@ export async function checkIfPlaceIndexExists(
     const result = await location.listPlaceIndexes().promise();
     return result?.Entries?.some((entry) => entry.IndexName === placeIndexName);
   } catch (error) {
-    logger.error(`checkIfPlaceIndexExists Error: ${error.message}`);
+    const typedError = error as { message: string };
+    logger.error(`checkIfPlaceIndexExists Error: ${typedError.message}`);
```

Comment:
errors natively have a `message` property. any reason for this change?

Reply:
Might just be a VSCode thing but typescript defaults the error variable to `unknown` and it shows this error

<img width="802" alt="Screenshot 2022-08-23 at 13 20 21" src="https://user-images.githubusercontent.com/1377253/186156857-c25b9c27-da06-427a-ab14-53e11131c295.png">

Changing it from an `unknown` to something else makes it angry
<img width="870" alt="Screenshot 2022-08-23 at 13 21 37" src="https://user-images.githubusercontent.com/1377253/186156958-398a8239-0337-4c9f-b2c0-16c15e46228e.png">

So creating a new variable for it worked the best for me. Although I guess I could cast it to an `Error` instead

Reply:
it didn't seem to cause an error beforehand. do you need to reconfigure VSC?

Reply:
Nope, must be a TS update 🤷

Reply:
but the error goes away if the error type annotation is `any`, so that will fix it but I'm not an `any` kind of TS developer 😄